ProjectFiles/Assets/Editor/ServerBuild.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+
+public class ServerBuild
+{
+    
+    [MenuItem("Build/Build Server")]
+    public static void BuildServer()
+    {
+        BuildPlayerOptions buildPlayerOptions = new BuildPlayerOptions();
+        buildPlayerOptions.scenes = new[] { "Assets/Scenes/ModelShowcase.unity" };
```

Comment:
Maybe we could make this an inspector interface? (If building from the editor). So we can add levels, although I guess we won't be doing that very often.

Reply:
Yeah fairs, it sounds like people are more likely to just build using `build.sh` in which case it would just be moving the hard-coding of the scenes to another location :thinking:. Once we have started to create the actual test level I doubt it will ever change after that.